protostar/commands/test/test_execution_environment.py

```diff
@@ -45,33 +49,40 @@ def contract_address(self):
 
 
 class TestExecutionEnvironment:
+    # pylint: disable=too-many-arguments
     def __init__(
         self,
         include_paths: List[str],
         forkable_starknet: ForkableStarknet,
         test_contract: StarknetContract,
         test_context: TestContext,
+        starknet_compiler: StarknetCompiler,
```

Comment:
We should rename it to `CairoCompilerFacade` at some point, current name doesn't make much sense (don't do it here)